styles.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  margin-top: 10px;
+}
+
+body {
+  background-image: url('./Images/header-shapes\ mobile.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+  margin: 10px 30px;
+}
+
+.logo {
+  height: 20px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  order: 0;
+  flex-grow: 0;
+  text-decoration: none;
+}
+
+.navbar {
+  display: block;
+  width: 18px;
+  height: 3px;
+  margin: 3px;
+  background-color: #6070ff;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 606px;
+  align-items: center;
+  padding-top: 50px;
+  padding-right: 20px;
+}
+
+.title {
+  height: 150px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: stretch;
+  color: #172b4d;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 30px;
+  margin-right: 20px;
+}
```

Comment:
- [ ] Kindly make sure to align your main title correctly in order to follow the provided Figma design. :wink: 

    - ### YOUR DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/205847493-e9231796-e452-4848-a07f-facd2bf45def.png)


    - ### FIGMA DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/205847667-e554b1fb-e72a-473e-96a8-ec9421030aaa.png)